core/src/avm2/multiname.rs

```diff
@@ -339,13 +324,17 @@ impl<'gc> Multiname<'gc> {
         }
     }
 
-    /// Indicates if this multiname matches any type in any namespace.
-    pub fn is_any(&self) -> bool {
+    /// Indicates if this multiname matches any type.
+    pub fn is_any_name(&self) -> bool {
         self.name.is_none()
-            && match self.ns {
-                NamespaceSet::Single(ns) => ns.is_any(),
-                NamespaceSet::Multiple(ns) => ns.iter().any(|ns| ns.is_any()),
-            }
+    }
+
+    /// Indicates if this multiname matches any namespace.
+    pub fn is_any_namespace(&self) -> bool {
+        match self.ns {
+            NamespaceSet::Single(ns) => ns.is_any(),
+            NamespaceSet::Multiple(ns) => ns.iter().any(|ns| ns.is_any()),
```

Comment:
(I think it's actually not allowed for namespace-sets to contain Any names - we could validate and use that maybe, but that's not needed in this PR)